share/picongpu/pypicongpu/schema/laser.DispersivePulseLaser.json

```diff
@@ -0,0 +1,169 @@
+{
+  "$id": "https://registry.hzdr.de/crp/picongpu/schema/picongpu.pypicongpu.laser.DispersivePulseLaser",
+  "description": "Specification of a dispersive Gaussian laser pulse with dispersion parameters",
+  "type": "object",
+  "properties": {
+    "wave_length_si": {
+      "type": "number",
+      "description": "wavelength of laser [m]",
+      "exclusiveMinimum": 0
+    },
+    "waist_si": {
+      "type": "number",
+      "description": "Waist of the Gaussian pulse at focus [m]",
+      "exclusiveMinimum": 0
+    },
+    "pulse_duration_si": {
+      "type": "number",
+      "description": "sigma of std. gauss for intensity (E^2) [s]",
+      "exclusiveMinimum": 0
+    },
+    "focus_pos_si": {
+      "description": "position of focus in total coordinate system, [m]",
```

Comment:
total or global?

Reply:
According to https://github.com/ComputationalRadiationPhysics/picongpu/wiki/PIConGPU-domain-definitions, you are right. Since there has never been a difference between total and global (for us/for me), I didn't even know we had a total coordinate system 😄